Mathlib/Algebra/Order/WithTop/Untop0.lean

```diff
@@ -65,6 +65,13 @@ lemma untopD_add [Add α] {a b : WithTop α} {c : α} (ha : a ≠ ⊤) (hb : b 
 lemma untop₀_add [AddZeroClass α] {a b : WithTop α} (ha : a ≠ ⊤) (hb : b ≠ ⊤) :
     (a + b).untop₀ = a.untop₀ + b.untop₀ := untopD_add ha hb
 
+@[simp]
+lemma untop₀_neg [AddCommGroup α] : ∀ a : WithTop α, (-a).untop₀ = -a.untop₀
```

Comment:
For `GroupWithZero`, can you also prove `untop₀_inv`?

Reply:
You shouldn't. I confused with `WithZero`, but it would be nice whether the corresponding `WithZero` lemma exists

Reply:
Still unsure. What do you want me to do?